tmt/steps/report/display.py

```diff
@@ -38,8 +38,18 @@ class ReportDisplay(tmt.steps.report.ReportPlugin[ReportDisplayData]):
     """
     Show test results on the terminal.
 
+    *As a tester I want to see test results in the plain text
+    form in my shell session.*
```

Comment:
I guess it would be ok to drop these user story texts here. They were needed for the `tmt story` to be defined. I think the description should now be sufficient. Similar for other `report` plugins.

Reply:
Done !